Userland/Libraries/LibGUI/GlyphMapWidget.h

```diff
@@ -10,6 +10,7 @@
 #include <LibGUI/AbstractScrollableWidget.h>
 #include <LibGUI/TextRange.h>
 #include <LibGfx/BitmapFont.h>
+#include <LibUnicode/UnicodeData.h>
```

Comment:
We can't rely on this existing because it won't be generated if the `ENABLE_UNICODE_DATABASE_DOWNLOAD` CMake option is disabled (for example, in Fuzzer builds).

Instead, if we need the `CodePointRange` type, that can just be put in the always-existing LibUnicode/CharacterTypes.h header instead.